app/livechat/client/views/app/tabbar/customerChatHistory.js

```diff
@@ -0,0 +1,155 @@
+import { Template } from 'meteor/templating';
+import moment from 'moment';
+import { ReactiveVar } from 'meteor/reactive-var';
+import './customerChatHistory.html';
+import './chatRoomHistoryItem.html';
+import './chatRoomSearchItem.html';
+import _ from 'underscore';
+
+import { APIClient } from '../../../../../utils/client';
+
+const ITEMS_COUNT = 50;
+Template.customerChatHistory.helpers({
+
+	hasChatHistory() {
+		// will return if user has any chatHistory or not
```

Comment:
We don't need comments in all helper methods.

Reply:
removed